src/main/java/com/adaptavist/tm4j/jenkins/extensions/JiraInstance.java

```diff
@@ -7,12 +7,11 @@
 import hudson.util.Secret;
 import org.apache.http.client.HttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
-import org.kohsuke.stapler.DataBoundConstructor;
 
 import java.io.File;
 import java.text.MessageFormat;
 
-public class JiraInstance {
+public class JiraInstance implements Instance {
```

Comment:
I'd call it `JiraServerInstance`

Reply:
Is this going to change the persisted XML file if we rename the class? I don't really know. Just wondering.

Reply:
Please see my previous comment :)